data/data/openstack/main.tf

```diff
@@ -53,9 +53,11 @@ module "masters" {
     var.openstack_master_extra_sg_ids,
     [module.topology.master_sg_id],
   )
-  root_volume_size = var.openstack_master_root_volume_size
-  root_volume_type = var.openstack_master_root_volume_type
-  server_group_id  = var.openstack_master_server_group_id
+  root_volume_size           = var.openstack_master_root_volume_size
+  root_volume_type           = var.openstack_master_root_volume_type
+  server_group_id            = var.openstack_master_server_group_id
+  additional_network_ids     = var.openstack_additional_network_ids
+  additional_security_groups = var.openstack_additional_security_groups
```

Comment:
Looks like we already have `openstack_master_extra_sg_ids` variable with the same purpose. It's added to the `master_sg_ids` variable above on line 52.